template/page-impact.html.jinja

```diff
@@ -8,119 +8,226 @@
-      <p class="mb-6">{% trans %}Bitcoin has historically been recognized as a store of value and Ether (ETH) for its smart contract capabilities.{% endtrans %}</p>
-      <p class="mb-6">{% trans %}XRP was designed to be optimal for global payments—it’s fast, cheap, scalable and energy-efficient. But the XRP Ledger is also increasingly popular for innovative blockchain use cases, like NFTs and DeFi.{% endtrans %}</p>
-      <p class="mb-6">{% trans %}The same characteristics that make XRP ideal for global transactions mean it’s also better for our environment. XRP is green by nature.{% endtrans %}</p>
+    <div class="col-md-6 offset-md-3 p-6-sm p-10-until-sm br-8 cta-card">
+      
+      <img src="./img/backgrounds/cta-community-purple.svg" class="cta cta-top-left">
+      <img src="./img/backgrounds/cta-calculator-green.svg" class="cta cta-bottom-right">
+      <div class="z-index-1 position-relative">
+        <div class="d-flex flex-column-reverse">
+          <h2 class="h4 h2-sm mb-10-until-sm mb-8-sm">{% trans %}Featured companies & projects running on the XRP Ledger.{% endtrans %}</h2>
+          <h6 class="eyebrow mb-3">{% trans %}Sustainable Projects{% endtrans %}</h6>
+        </div>
+        <p class="mb-10">{% trans %}Learn more about companies and developers who are using the XRP Ledger to solve interesting problems efficiently and sustainably.{% endtrans %}</p>
+        <a class="btn btn-primary btn-arrow" href="/">{% trans %}See More{% endtrans %}</a>
+      </div>
     </div>
-  </section>
+</section>
 
+<!-- Connect -->
+<section class="container-new py-26">
+  <!-- flex. Col for mobile. Row for large. on large align content to the center  -->
+  <div class="d-flex flex-column flex-lg-row align-items-lg-center mr-lg-4">
+
+      <div class="order-1  mb-4 pb-3 mb-lg-0 pb-lg-0 col-lg-6 px-0 p-lg-3">
+          <div class="d-flex flex-column-reverse py-lg-3">
+              <h3 class="h4 h2-sm">{% trans %}How can developers connect and contribute?{% endtrans %}</h3>
+          </div>
+          <p class="py-lg-3 mb-2 longform" style="max-width: 520px;">{% trans %}If sustainability, security and decentralization are important to you, you’re invited to join the XRPL community in its quest to build a better world. Here are some ways to get involved:{% endtrans %}</p>
+
+          <div class="d-none d-lg-block py-lg-3">
+              <a class="btn btn-primary btn-arrow" href="/contribute.html">{% trans %}Join the Community{% endtrans %}</a>
+          </div>
+      </div>
+
+      <div class="order-2 col-lg-6 px-0 pl-lg-3">
+          <div class="row align-items-center m-0 connect-list"> 
+              <!-- connect list -->
+              <div class="col-12 col-lg-6 p-0 pr-3">
+                  <div class="px-lg-3 pb-3">
+                      <img id="connect-01" src="data:," />
+                      <div class="pt-3">
+                          <h6 class="mb-3">Blog</h6>
```

Comment:
I think this whole list is missing `{% trans %}` tags as well

Reply:
added trans tag for text